packages/terra-date-picker/docs/DatePickerField.md

```diff
@@ -0,0 +1,59 @@
+# Terra Date Picker Field
+
+[terra-date-picker](https://github.com/cerner/terra-framework/tree/master/packages/terra-date-picker) component that is wrapped inside a
+[terra-form-field](https://github.com/cerner/terra-core/tree/master/packages/terra-form-field) component.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-date-picker`
+  - `yarn add terra-date-picker`
+
+## Usage
+
+```jsx
+import React from 'react';
+import Base from 'terra-base';
+import DatePickerField from 'terra-date-picker/lib/DatePickerField';
+
+//Renders a date picker
+<Base locale='en'>
+  <DatePickerField
+    datePickerId="picker-id"
+    minDate={'2017-04-01'}
+    maxDate={'2017-04-30'}
+    excludeDates={['2017-04-10', '2017-04-20']}
+    filterDate={isWeekday}
+    name="admission-date"
+    label="Admission Date"
+    required
+  />
+</Base>
+
+
+const isWeekday = (date) => {
+  const day = date.day();
```

Comment:
#590 changes the parameter in filterDate from a moment object to be an ISO string. This should be updated to something like [this](https://github.com/cerner/terra-framework/pull/590/files#diff-4d64f16fdc0d286ca4cfba4a4e10601b). If we wanted to, we could just remove the illustration of filterDate in this example code to minimize maintenance.

Reply:
+1 to removing hard coded example